dali/kernels/signal/dct/dct.h

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_SIGNAL_DCT_DCT_H_
```

Comment:
Perhaps this file should be called `dct/table.h` or sth like this - calling it "dct.h" suggests that it's _the_ file one should include to use the functionality.

Reply:
renamed to table.h